src/test/java/com/google/step/youtube/YouTubeInfoScraperTest.java

```diff
@@ -164,17 +117,10 @@ public void scrapePromoCodesFromPlaylist_nonExistentUploadId() throws IOExceptio
 
     @Test
     public void scrapePromoCodesFromPlaylist_oneItem() throws IOException {
-        PlaylistItemListResponse testPlaylistResponse = new PlaylistItemListResponse();
-        String description = "Get 20% off your first monthly box and enter the code RO at checkout!";
-        testPlaylistResponse.setItems(Arrays.asList(new PlaylistItem()
-            .setSnippet(new PlaylistItemSnippet().setDescription(description).setPublishedAt(new DateTime(DATE))
-            .setTitle(VIDEO_TITLE).setResourceId(new ResourceId().setVideoId(VIDEO_ID)))));
+        PlaylistItemListResponse testPlaylistResponse = ytClientBuilder.newBasicPlaylistResponse();
         when(mockListPlaylistItems.execute()).thenReturn(testPlaylistResponse);
```

Comment:
Could move the declaration on line 120 to be inlined in line 121 so a new variable doesn't have to be declared.